models/Document/Editable/Link.php

```diff
@@ -351,7 +351,7 @@ public function setDataFromResource(mixed $data): static
         $fieldsToExclude = ['path'];
         foreach($this->data as $key => $value) {
             if(!in_array($key, $fieldsToExclude)) {
-                $this->data[$key] = SecurityHelper::sanitizeHtmlAttributes($value);
+                $this->data[$key] = SecurityHelper::sanitizeHtmlAttributes((string) $value);
```

Comment:
Couldn't we just check if the value is a string and then call the SecurityHelper or is the cast necessary here?

Reply:
we also casted the value in 10.6

Reply:
Hmm, yeah maybe it was not the best choice for 10.6 either, but then let's keep it this way.